example/test_problem/ELBDM/RestrictionMismatch/README.md

```diff
@@ -0,0 +1,25 @@
+# Demonstration of phase mismatch after refinement and restriction
+
+## Explanation 
+This test problem demonstrates that refining and restricting the wave function in ELBDM leads to a mismatch of the phase on the coarse levels when the option `OPT__INT_PHASE` is on.
+Further, it shows that the new option `OPT__RES_PHASE` remedies the mismatch immediately after restriction, but does not prevent the formation of artifacts at the coarse-fine boundary. 
+It uses the `GaussianWavePacket` test with suitable initial parameters and refines a region at the center of the Gaussian wave packet. 
+
+## Execution
+To run it, compile a version of GAMER with `MODEL = ELBDM` in `SERIAL` mode and put the executable in the test problem directory.
+Execute `./compare_restriction.sh` (possibly after making it executable via `chmod +x compare_restriction.sh`).
```

Comment:
`sh ./compare_restriction.sh` should work so no need to execute `chmod +x compare_restriction.sh`, right?

Reply:
Got it!